cloudant-client/src/main/java/com/cloudant/client/api/views/Key.java

```diff
@@ -170,6 +192,26 @@ private ComplexKey(JsonArray json) {
             return addObjects(numberKeys);
         }
 
+        public Key.ComplexKey add(EndKey endKey) {
```

Comment:
As you have gone down the method route then the `END_KEY` constant need not be exposed and the method doesn't have to take an argument.
e.g. it could just be `end()` (although that naming could give the impression it needs to always be called) or `addHigh()` or whatever